build/Dockerfile.openshift

```diff
@@ -0,0 +1,18 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+WORKDIR /go/src/github.com/nmstate/kubernetes-nmstate
+COPY . .
+RUN GO111MODULE=on go build --mod=vendor -o build/_output/bin/kubernetes-nmstate cmd/manager/main.go
+
+FROM centos:8
+
+RUN yum install -b -y dnf-plugins-core && \
+#    yum copr enable -y networkmanager/NetworkManager-1.22 && \
+#    yum copr enable -y nmstate/nmstate-0.2 && \
+    yum install -b -y nmstate iproute iputils && \
+    yum remove -y dnf-plugins-core && \
+    yum clean all
+
+COPY --from=builder /go/src/github.com/nmstate/kubernetes-nmstate/build/_output/bin/kubernetes-nmstate  /usr/bin/
+COPY --from=builder /go/src/github.com/nmstate/kubernetes-nmstate/build/bin   /usr/bin/
+
+ENTRYPOINT ["/usr/bin/kubernetes-nmstate"]
```

Comment:
Don't we need to `LABEL` the dockerfile? https://github.com/openshift/cluster-network-operator/blob/master/Dockerfile.rhel7#L16

Reply:
I will need to find it, but I think I saw something about needing to wait to do that until something happens on the CVO side maybe? Let me see if I can dig up that reference.

Reply:
I can't find it. Ignore me. :)